dist/dist.py

```diff
@@ -33,10 +32,19 @@ def all_c_files():
         if fnmatch.fnmatch(file, '*_inline.h'):
             yield file
 
+# all_cpp_files --
```

Comment:
That would be nice! `src/btree` is big, so it would be nice to break it up this way. After that, we can modify function `infer_module` in `function.py` to account for this.